src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -519,6 +531,15 @@ private void Run()
                 }
             }
 
+            if (this.IsParameterBound(c => c.ScaleInPolicy))
+            {
+                if (vScaleInPolicy == null)
```

Comment:
I thinks this is always true.

Reply:
This is from generated code.  The reason for null checking is if there are multiple properties under the same object, we don't want to re-create the object.